Lib/test/pythoninfo.py

```diff
@@ -924,9 +924,12 @@ def collect_windows(info_add):
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.PIPE,
                                 text=True)
```

Comment:
```suggestion
                                text=True,
                                env={
                                    "SystemDrive": os.environ["SystemDrive"],
                                    "SystemRoot": os.environ["SystemRoot"],
                                    "Path": r"{0}\system32;{0}".format(os.environ["SystemRoot"]),
                                    "PATHEXT": os.environ["PATHEXT"],
                                })
```

Just an idea to try and avoid issues with GHA's environment? You'll also need an `import os` as there isn't one in this file yet.